packages/airnode-deployer/terraform/gcp/providers.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = "~> 4.7"
+      version = "~> 4.54"
```

Comment:
Downgrade?

Reply:
I thought so as well at the first glance but it's actually from `7` to `54`. It's not `5.4`.